build/server/middlewares/authentication.js

```diff
@@ -1,70 +0,0 @@
-// Generated by CoffeeScript 1.10.0
```

Comment:
I though that with only keep build directory on `master` branch to keep all dev branches clean and prevent commit mess with useless diff concerning build folder. Nope?